frontend/src/layout/navigation-3000/components/ProjectTree/projectTreeLogic.tsx

```diff
@@ -26,20 +27,25 @@ export const projectTreeLogic = kea<projectTreeLogicType>([
             ['aggregationLabel', 'groupTypes', 'groupsAccessStatus'],
             featureFlagLogic,
             ['featureFlags'],
+            teamLogic,
+            ['currentTeamName'],
         ],
     }),
     actions({
         loadSavedItems: true,
         loadUnfiledItems: (type?: FileSystemType) => ({ type }),
         addFolder: (folder: string) => ({ folder }),
         deleteItem: (item: FileSystemEntry) => ({ item }),
-        moveItem: (oldPath: string, newPath: string) => ({ oldPath, newPath }),
+        moveItem: (oldFilePath: string, newFilePath: string) => ({ oldFilePath, newFilePath }),
         queueAction: (action: ProjectTreeAction) => ({ action }),
         removeQueuedAction: (action: ProjectTreeAction) => ({ action }),
         applyPendingActions: true,
         createSavedItem: (savedItem: FileSystemEntry) => ({ savedItem }),
         updateSavedItem: (savedItem: FileSystemEntry) => ({ savedItem }),
         deleteSavedItem: (savedItem: FileSystemEntry) => ({ savedItem }),
+        setExpandedFolders: (folders: string[]) => ({ folders }),
+        setActiveFolder: (folder: string | null) => ({ folder }),
+        setLastViewedPath: (path: string) => ({ path }),
```

Comment:
Having `X` and `setX` pairs is an anti pattern in Kea. It's better to just have actions that are, well, actions: `toggleFolder`, `selectFolder`, etc, and then have reducers change the field's contents accordingly.

More info about the anti pattern here: https://keajs.org/docs/core/reducers